sentry/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG SENTRY_IMAGE
+FROM ${SENTRY_IMAGE}
+
+COPY . /usr/src/sentry
```

Comment:
Why should we copy all contents of self-hosted to sentry image?

Reply:
It is just copying the contents of `sentry`, since that is the build context defined in `docker-compose.yml`. It is necessary to copy the entire folder because `sentry/requirements.txt` and `sentry/enhance-image.sh` are optional. It is not possible to copy single files conditionally in a `Dockerfile`.

Reply:
> It is just copying the contents of `sentry`,

Oops, right.

> since that is the build context defined in `docker-compose.yml`. It is necessary to copy the entire folder because `sentry/requirements.txt` and `sentry/enhance-image.sh` are optionally present and influence the build.

Right.

> It is not possible to copy single files conditionally in a `Dockerfile`.

And right again!